src/org/usfirst/frc/team2706/robot/commands/teleop/GearHandler.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class GearHandler extends Command {
+
+    private DoubleSolenoid solenoid = new DoubleSolenoid(4,5);
+    private Joystick stick ;
+    private boolean closed;
```

Comment:
Usually we put spaces between each initializing variable, just for readability, also comment these